src/sonic-yang-mgmt/setup.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
+from setuptools import setup, find_packages
+from setuptools.command.build_py import build_py
+from os import system
+from sys import exit
+
+
+# important reuirements parameters
+build_requirements = ['../../target/debs/stretch/libyang0.16_0.16.105-1_amd64.deb',
```

Comment:
https://github.com/zhenggen-xu/sonic-buildimage/pull/4 was checked in, please update the libyang version here to reflect that change.

Reply:
Done.